test/views/question_views/simpleQuestion.html

```diff
@@ -9,7 +9,7 @@ <h1>Question 1</h1>
   {% endif %}
 
   {{ fields.name }}
-  <input name="{{ fields.name.id }}"
+  <input name="{{ fields.name.name }}"
```

Comment:
value?

Reply:
The name of the field that the input should be posted to on a POST.